src/pages/resultsView/plots/PlotsTabUtils.tsx

```diff
@@ -132,22 +130,47 @@ export interface IScatterPlotSampleData {
     copyNumberAlterations: AnnotatedNumericGeneMolecularData[];
 }
 
-export interface IScatterPlotData extends IScatterPlotSampleData, IBaseScatterPlotData {};
+export interface IThresholdDataPoint {
+    thresholdType?:'>'|'<'|undefined;
+}
+export interface IThresholdsDataPoint {
```

Comment:
good but threshold and thresholds are too similar to the eye. better 1DThreshold and 2DThreshold

Reply:
Interfaces were renamed to `IDataPoint1D` and `IDataPoint2D`.